CHANGELOG.md

```diff
@@ -28,6 +28,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (`JxlDecoderDCOutBufferSize` and `JxlDecoderSetDCOutBuffer`) and will be
   removed in the next release.
 
+## [0.5] - 2021-07-??
+### Changed
+ - Add JxlDecoderRewind and JxlDecoderSkipFrames to decoder API to skip more
+   efficiently to earlier animation frames.
+ - Add JxlDecoderSetPreferredColorProfile to decoder API to choose color profile
+   in certain circumstances.
+ - Deprecate JxlDecoderDCOutBufferSize, JxlDecoderSetDCOutBuffer,
```

Comment:
This goes in the Removed section and I think it is already there.

Reply:
Fixed